### DISCUSSION TITLE
Demography model

### DISCUSSION THREAD

**Discussion by vmiropina** [tskit-dev/msprime#1996] [Ideas]
Hello, 

I would like to know if there is a way to add arbitrary demography. 
For example, instead of exponential growth I would like to have logistic or Gompertz or this type of functions.
Is that possible ?

**Comment by jeromekelleher** [comment]
Hi @vmiropina :wave:

This is a great question! In principle these types of population growth functions could be accommodated, but they aren't implemented in msprime. It should just be a case of adding some extra parameters and implementing the required functions [here](https://github.com/tskit-dev/msprime/blob/0807f6d44eb91c9503f27b5c2c19bc2ce478b96d/lib/msprime.c#L87), but it would be a significant project to actually make the changes.

**Comment by vmiropina** [comment]
Thank you very much for your answer !
